### DISCUSSION TITLE
Plugin for BGS Activity Tracking

### DISCUSSION THREAD

**Discussion by aussig** [EDCD/EDMarketConnector#1272] [Plugin Development]
Hi team, superb work on EDMC, I use it continuously and even knew Jonathan a bit though his X-Plane stuff, though I believe he's moved on now from both X-Plane and E:D.  Anyway, I hope this is the right place for letting you know about a plugin that's not on your [wiki plugins page](https://github.com/EDCD/EDMarketConnector/wiki/Plugins):

I needed a little tool to track BGS activity in E:D and thought it would be good to 'give back' to the community a little and create an EDMC plugin for this.  After a bit of hunting I found that [tezw21](https://github.com/tezw21) had already gone some way towards what I wanted with his [BGS-Tally-v2.0 plugin](https://github.com/tezw21/BGS-Tally-v2.0). So I've forked his work and extended it to do some additional stuff that I needed. Mine is here if you wanted to take a look: [BGS-Tally](https://github.com/aussig/BGS-Tally).  Note that as part of the fork I renamed it to remove v2.0 as I didn't like having a version number in the title, but this is definitely based on tezw21's v2.0, not his original plugin which was less featured. I also removed Python 2 support.

It mostly does what I want it to now, but do have a few [plans for future features](https://github.com/aussig/BGS-Tally/labels/enhancement), so I expect there will be occasional updates released in future.

Thanks again for your work!

**Comment by Athanasius** [comment]
I was intending to find some time to install this myself *and* play the game some with it, but that never happened.  So instead I'm going to trust it works, especially as you have a nice set of Wiki documentaiton for it.  Added to our wiki Plugins page.

**Reply by aussig** [reply]
Thank you very much, that's great.

We have quite a few users in our squadron who find it useful and have highlighted a few bugs that have been fixed.

If there is any concern about what it's doing, most of it is just keeping local counts of activity on the user's machine. The only network connections are a) To get the latest tick from elitebgs.app and b) to send a report to a Discord server / channel of the user's choice.  The latter is fully under the user's control, no data is sent anywhere if they don't set up the Disc integration themselves, and a button must also be pressed to send any data. 

If anyone has any questions, just ping me here, or raise an issue on the GitHub project.
